README.md

```diff
@@ -4,79 +4,137 @@ OpenEuropa Paragraphs
 [![Build Status](https://drone.fpfis.eu/api/badges/openeuropa/oe_paragraphs/status.svg?branch=master)](https://drone.fpfis.eu/openeuropa/oe_paragraphs)
 [![Packagist](https://img.shields.io/packagist/v/openeuropa/oe_paragraphs.svg)](https://packagist.org/packages/openeuropa/oe_paragraphs)
 
-This module integrates the
-[ECL](https://github.com/ec-europa/europa-component-library) - the component
-library of the European Commission - with the
-[Paragraphs](https://www.drupal.org/project/paragraphs) module.
+This module provides a number of Paragraph types that are based on a select number of components integrating [ECL](https://github.com/ec-europa/europa-component-library),
+the component library of the European Commission.
+These are intended to be used on landing pages of various sites of the European Commission.
 
-It provides a number of Paragraph types that are based on a select number of
-ECL components. These are intended to be used on landing pages of various sites
-of the European Commission.
+**Table of contents:**
 
+- [Installation](#installation)
+- [Development setup](#development-setup)
+- [Contributing](#contributing)
+- [Versioning](#versioning)
+
+## Installation
+
+The recommended way of installing the OpenEuropa Authentication module is via [Composer][1].
+
+```bash
+composer require openeuropa/oe_paragraphs
+```
+
+### Enable the module
+
+In order to enable the module in your project run:
+
+```bash
+./vendor/bin/drush en oe_paragraphs
+```
 
 ## Development setup
 
 You can build a test site by running the following steps.
 
 * Install all the composer dependencies:
 
-```
-$ composer install
+```bash
+composer install
 ```
 
 * Customize build settings by copying `runner.yml.dist` to `runner.yml` and
 changing relevant values.
 
 * Setup test site by running:
 
-```
-$ ./vendor/bin/run drupal:site-setup
+```bash
+./vendor/bin/run drupal:site-setup
```

Comment:
This should be removed like in other repositories we are now updating.

Reply:
Fixed.